exchanges/bitmex/bitmex_test.go

```diff
@@ -868,8 +868,8 @@ func TestWSOrderbookHandling(t *testing.T) {
       ]
     }`)
 	err = b.wsHandleData(pressXToJSON)
-	if err != nil {
-		t.Error(err)
+	if err == nil {
+		t.Error("delete should already have occurred above")
```

Comment:
I think something like this captures more issues:
```
if err != nil && err.Error() != "update cannot be deleted id: 17999995000 not found" {
		t.Error(err)
	}
	if err == nil {
		t.Error("expecting error")
	}
```
It ensures the error we're expecting is the one we receive